src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -158,6 +158,7 @@ internal class ResourceNames
             internal const string TCE_DbConnectionString_AttestationProtocol = @"Specifies an attestation protocol for its corresponding enclave attestation service.";
             internal const string TCE_DbConnectionString_IPAddressPreference = @"Specifies an IP address preference when connecting to SQL instances.";
             internal const string SqlConnection_ServerProcessId = @"Server Process Id (SPID) of the active connection.";
+            internal const string SqlCommandBuilder_DataAdapter = @"The DataAdapter for which to automatically generate SqlCommands.";
```

Comment:
It's not required. Please revert it.

Reply:
Without this it does not compile with the merged [ResDescriptionAttribute(StringsHelper.ResourceNames.SqlCommandBuilder_DataAdapter)]